File: BEmu/Types/Rules.cs
//------------------------------------------------------------------------------
// <copyright project="BEmu_csh" file="Types/Rules.cs" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Bloomberglp.Blpapi.Types
{
    internal static class Rules
    {
        public static bool IsSecurityError(string security)
        {
            return security == null || security.StartsWith("Z", StringComparison.OrdinalIgnoreCase);
        }

        public static bool IsBadField(string field)
        {
            return field == null || field.StartsWith("Z", StringComparison.OrdinalIgnoreCase);
        }
    }
}

File: BEmuJava/bemu/src/main/java/com/bemu/BEmu/types/Rules.java
//------------------------------------------------------------------------------
// <copyright project="BEmuJava" file="BEmu.Types.Rules.java" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

package com.bemu.BEmu.types;

public class Rules
{
	public static boolean isSecurityError(String security)
	{
		if(security == null)
			return true;
		else 
			return security.startsWith("Z") || security.startsWith("z"); 
	}
	
	public static boolean isBadField(String field)
	{
		if(field == null)
			return true;
		else 
			return field.startsWith("Z") || field.startsWith("z"); 
	}
}
